samples/sdl_img/Makefile

```diff
@@ -0,0 +1,10 @@
+XBE_TITLE = sdl_img
+GEN_XISO = $(XBE_TITLE).iso
+SRCS += $(wildcard $(CURDIR)/*.c)
+NXDK_DIR = $(CURDIR)/../..
+NXDK_SDL = y
+include $(NXDK_DIR)/Makefile
+
+$(XBE_TITLE).iso: $(OUTPUT_DIR)/testimg.jpg
```

Comment:
This is an interesting way of getting the files into the ISO. However, shouldn't this use `$(GEN_XISO)`?

We also do this differently in the SDL_ttf sample (and maybe elsewhere).

Reply:
> This is an interesting way of getting the files into the ISO. However, shouldn't this use `$(GEN_XISO)`?

That's probably better, yes. I changed it.
> We also do this differently in the SDL_ttf sample (and maybe elsewhere).

Afaik SDL_ttf is the only other sample where we need additional files in the ISO. I wasn't entirely happy with the solution user there, though - it doesn't properly use dependencies, so it won't rebuild the ISO when the file changes, and it looks like it might cause race conditions, too (ISO creation might run parallel to the cp_font rule).

Reply:
Please create an issue about this. It should be more consistent in the future. I feel like we have to improve this.

*(I also hope we'll move the data files into a folder like "samples/common/data" eventually, to avoid duplication of things like audio files for OpenAL, SDL Audio, xaudio, ... samples; or images for SDL_image, libpng, ... samples; or fonts in general)*

Reply:
(Handled in issue 322)